BUILDING.md

```diff
@@ -163,7 +163,8 @@ $ [sudo] make install
 
 Prerequisites:
 
-* [Python 2.6 or 2.7](https://www.python.org/downloads/)
+* [Python 2.6 or 2.7](https://www.python.org/downloads/)  
+  *Note*: It is recommanded to include the "Add python.exe to path" option
```

Comment:
So maybe we should add a note about the alternative ("add it to `%Path%` manually") if the #13882 has any chance to not cover all the cases.